src/apis/notification/index.ts

```diff
@@ -0,0 +1,8 @@
+import authorizedAxiosInstance from '~/axios'
+import { ApiResponse } from '~/interfaces/ApiResponse'
+import { NotificationInterface } from '~/interfaces/notification'
+import { API_ROOT } from '~/utils/constant'
+
+export const getNotificationByUserIdAPI = async (): Promise<ApiResponse<NotificationInterface[]>> => {
+  return await authorizedAxiosInstance.get(`${API_ROOT}/v1/notification`)
+}
```

Comment:
`export const getNotificationByUserIdAPI = (): Promise<ApiResponse<NotificationInterface[]>> => 
  authorizedAxiosInstance.get(`${API_ROOT}/v1/notification`);`
  
  It no need to return await in the async function